frontend/public/components/utils/_hint-block.scss

```diff
@@ -1,5 +1,6 @@
 .co-hint-block {
-  background-color: $pf-color-blue-50;
-  color: $pf-color-blue-600;
-  padding: 10px 15px 10px;
+  background-color: var(--pf-global--BackgroundColor--400);
+  border: 1px solid var(--pf-global--BorderColor--300);
```

Comment:
```suggestion
  background-color: var(--pf-global--palette--blue-50);
  border: 1px solid var(--pf-global--palette--blue-100);
```

Reply:
I assumed we'd use dark theme rules based on the PF `pf-c-hint` no?
https://github.com/patternfly/patternfly/blob/357331fa9b2b3da8af40ae188b7bd9d45319f3c0/src/patternfly/components/Hint/themes/dark/hint.scss#L1-L6
<img width="1146" alt="Screen Shot 2022-03-08 at 6 03 33 PM" src="https://user-images.githubusercontent.com/1874151/157341129-903c8a4b-0674-454b-a62d-b9c441a512fa.png">

Reply:
Oh yeah, definitely. I'd add this to support both cases:

 ``` 
  :where(.pf-theme-dark) & {
    background-color: var(--pf-global--BackgroundColor--400);
    border: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--300);
  }
```